packages/synapse-react-client/src/components/Plot/Plot.stories.ts

```diff
@@ -1,11 +1,11 @@
 import { Meta, StoryObj } from '@storybook/react'
-import PlotlyWrapper from '../PlotlyWrapper'
+import Plot from './Plot'
 import rawData from '../../mocks/distribution_data.json'
 import Plotly from 'plotly.js-basic-dist'
 
 const meta = {
-  title: 'Components/PlotlyWrapper',
-  component: PlotlyWrapper,
+  title: 'Components/PlotlyPlot',
```

Comment:
Update story to just use the Plot component instead of the unused PlotlyWrapper component